mne_bids/mne_bids.py

```diff
@@ -333,9 +367,9 @@ def raw_to_bids(subject_id, task, raw_file, output_path, session_id=None,
     events_data : str | array | None
         The events file. If a string, a path to the events file. If an array,
         the MNE events array (shape n_events, 3). If None, events will be
-        inferred from the stim channel using `find_events`.
-    event_id : dict
-        The event id dict
+        inferred from the stim channel using `mne.find_events`.
+    event_id : dict | None
```

Comment:
since we haven't made a release yet, we could change this arg to be `trial_type` as you have in one of your helper functions if you think that would be clearer

Reply:
It's a dilemma ... one way to do it is closer to MNE, the other is closer to BIDS, and here we are in the middle of MNE-BIDS ;-) I am 50-50 on this. Someone else decide, please.

Reply:
i'm fine leaving it MNE-like here since users will be familiar with it. we can keep the internals more BIDS like as it is currently